src/vaibhav/MinMaxNumber.java

```diff
@@ -0,0 +1,75 @@
+package vaibhav;
+
+public class MinMaxNumber {
+	
+	int minNumber(int[] arr) {		
+		int min=arr[0];
+		for (int index = 0 ;index < arr.length; index++){			
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files